### PR TITLE
Add new check attributes

### PR DESCRIPTION
**Pull request by echlebek**
Added max_output_size and discard_output attributes

Signed-off-by: Eric Chlebek <eric@sensu.io>

## Description
These attributes have been added to the core/v2 API. Associated PR: https://github.com/sensu/sensu-go/pull/2653

### CODE REVIEW

File: content/sensu-go/5.2/reference/checks.md [hunk 1]
```diff
@@ -413,6 +413,22 @@ required     | required if `splay` attribute is set to `true`
 type         | Integer
 example      | {{< highlight shell >}}"splay_coverage": 90{{< /highlight >}}
 
+### Check output truncation attributes
+
+|max_output_size  | |
+-------------|-------
+description  | Maximum output size, in bytes, that stored check outputs will have. When set to a non-zero value, check output will be truncated before being stored to etcd, if the output size is larger than this value. Filters, handlers, and mutators will still get access to the full check output.
```
Comment (apaskulin) [inline content/sensu-go/5.2/reference/checks.md]:
```suggestion
description  | Maximum size, in bytes, of stored check outputs. When this attribute is set to a non-zero value, the Sensu backend truncates check outputs larger than this value before storing to etcd. `max_output_size` does not affect data sent to Sensu filters, mutators, and handlers.
```

File: content/sensu-go/5.2/reference/checks.md [hunk 2]
```diff
@@ -413,6 +413,22 @@ required     | required if `splay` attribute is set to `true`
...
+required     | false
+type         | Integer
```
Comment (apaskulin) [inline content/sensu-go/5.2/reference/checks.md:422]:
```suggestion
type         | Integer greater than zero
```

File: content/sensu-go/5.2/reference/checks.md [hunk 3]
```diff
@@ -413,6 +413,22 @@ required     | required if `splay` attribute is set to `true`
...
+example      | {{< highlight shell >}}"max_output_size": 1024{{< /highlight >}}
+
+|discard_output  | |
+-------------|------
+description  | Discard check output after extracting metrics on the agent. No check output will be sent to the backend.
```
Comment (apaskulin) [inline content/sensu-go/5.2/reference/checks.md]:
```suggestion
description  | Discard check output after extracting metrics. No check output will be sent to the Sensu backend.
```

**Review by apaskulin — APPROVED** [review]
Thanks for opening this PR! I made a few wording suggestions, but otherwise 💯

File: content/sensu-go/5.2/reference/checks.md (hunk 2 above)
Comment (echlebek) [reply content/sensu-go/5.2/reference/checks.md:422]: It can also be zero :)

**Pull request merged**